app/browser/reducers/shareReducer.js

```diff
@@ -0,0 +1,21 @@
+/* This SourceCode Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const appConstants = require('../../../js/constants/appConstants')
+const {makeImmutable} = require('../../common/state/immutableUtil')
+const {simpleShareActiveTab} = require('../share')
+
+const shareReducer = (state, action) => {
```

Comment:
Food for thought on this one- what other types of sharing do we think we'll have?  Could this possibly handle things like `Submit Feedback` in the Help Menu, etc?